packages/analytics/client/README.md

```diff
@@ -329,3 +329,11 @@ The `_meta` adds the invaluable information of a `description` and whether a fie
 It can be attached to any schema definition as seen in the examples above. For high-order types, like arrays or objects, the `_meta` field is optional. For first-order types, like numbers, strings, booleans or `pass_through`, the `_meta` key is mandatory.
 
 The field `_meta.optional` is not required unless the schema is describing an optional field. In that case, `_meta.optional: true` is required. However, it's highly encouraged to be explicit about declaring it even when the described field is not optional.
+
+### Schema Validation
+
+Apart from documentation, the schema is used to validate the payload during the dev-cycle. 
+This way it adds an extra layer of confidence over the data to be sent. 
+The validation, however, is disabled in production because users cannot do anything to fix the bug after it is released. Additionally, receiving _buggy_ events can be considered an additional insight into how our users use our products. For example: the buggy event can be caused by a user following an unexpected path in the UI.
```

Comment:
My intention with this paragraph was to explain why we only want to validate events in dev mode and never in production.

In production, it's _OK_ to get malformed events because that highlights an unexpected bug in our logic or a corner case we didn't account for and we should allow the event through so we know about it. A good example is the [file upload you fixed a couple of months ago](https://github.com/elastic/kibana/pull/125180): if we had EBT back then, we'd report the filename and import options. The developer tests didn't try hitting the import button without selecting a file, so the EBT validation in dev-mode would have accepted filename as a string. However, in production, getting `filename` as undefined or `null`, would have highlighted the issue to us via telemetry. If we validate and skip the events in production, we miss those insights.